### PR TITLE
Adding the Serbian language to the application

### PR DESCRIPTION
**Pull request by branislavrt4316**
Hello,

I made a few changes to add the Serbian language to your app's options.

Changes are made to the following documents:
1. AnswerDto.cs
2. QuestionDto.cs
3. RuleDto.cs
4. Rule01.cs
5. Rule02.cs

New documents are added:
1. srb.json

In the next few days, I will make changes to the files Rule03.cs to Rule16.cs

Best regards,
Branislav Vujanovic

### CODE REVIEW

**Comment by KristofGilis** [comment]
@branislavrt4316

Already thank you for collaborating. Let me know when you finish the translations and I can put them on the website. 😉

**Comment by branislavrt4316** [comment]
Dear Kristof, everything is ready to upload the Serbian translation to the site! :)

File: i18n/srb.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    },
+    "rules_card": {
+      "button": "Иди на правила",
+      "content": "Идите на листу свих правила. Одавде можете добити сва питања у вези са изабраним правилом.",
+      "title": "Правила"
+    },
+    "subtitle": "Проверите своје знање о правилима игре. <br> Питања о ИХФ правилима игре која важе од <b>1. Јула 2024.</b>",
+    "title": "Добродошли"
+  },
+  "languages": {
```
Comment (KristofGilis) [inline i18n/srb.json:24]: Here you still need to add "serbian" ;-)

**Comment by KristofGilis** [comment]
@branislavrt4316 

Thanks for collaborating. I only add a really small comment. I try to add the Serbian translations asap to the website. Normally at the latest at the end of the week ;-)

**Review by KristofGilis — APPROVED** [review]

**Comment by branislavrt4316** [comment]
You are welcome. When a translation is ready on the website just let me know! ;)

**Comment by KristofGilis** [comment]
@branislavrt4316 
I updated the website and the Serbian version is now available ;)

**Pull request merged**